site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -149,13 +149,13 @@
       alert("This browser does not support desktop notification");
     }
     else if (Notification.permission === "granted") {
-      new Notification("Notifications have been enabled. You will be notified only when somebody joins the queue and there was nobody previously waiting for help.");
+      new Notification("Notifications have been enabled. You will be notified only if you are not helping a student and a student is now waiting for help when there was previously nobody waiting for help.");
```

Comment:
Suggestions of making it shorter:
I think it's enough to say ```You will be notified when a student joins an empty queue```, but if we want to be specific with the wording and cover the edge case of paused students, we can maybe say ```You will be notified when a student joins a previously inactive queue.```

Reply:
I actually like the phrasing of "previously inactive queue" and I'll try to work that in there. 

Although, the reason I removed the phrase "when a student joins" is because the notification isn't only triggered when a student joins. It can also be triggered when a student un-pauses. This is why I changed that phrasing, so that it could be clear when the notification would be sent. But if you think this specificity is unnecessary and the previous phrasing works well, we can easily switch it back.